smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/TomeeSmokeTest.groovy

```diff
@@ -5,17 +5,20 @@
 
 package org.hypertrace.agent.smoketest
 
+import spock.lang.Ignore
+
 import java.time.Duration
 import org.testcontainers.containers.wait.strategy.Wait
 import org.testcontainers.containers.wait.strategy.WaitStrategy
 
+@Ignore
```

Comment:
is tomee also failing? If so create an issue

Reply:
the v.15.0 release did not have test for `tomee` . However it is present in latest code. 
The the image matrix that is currently published is based on v.15.0 release so tomee test container images are missing in docker hub. 
if you look for changes in current PR in file: smoke-tests/matrix/build.gradle
I have added config to publish the same. 
Once this PR is merged, images will be published and I will create a follow up PR to enable this with new image tags.